Sources/PulseUI/Features/Console/List/ConsoleListView.swift

```diff
@@ -168,7 +157,7 @@ private struct _ConsoleListView: View {
             }
             Divider()
             ScrollViewReader { proxy in
-                List(selection: $selection) {
+                List(selection: $router.selection) {
```

Comment:
By linking directly to the router, we can remove local state and keep the view in sync with the router at all times.